pkg/util/readfile.go

```diff
@@ -0,0 +1,15 @@
+package util
+
+import (
+	"io/ioutil"
+	"strings"
+)
+
+func LoadWhitelistPods(path string) ([]string, error) {
```

Comment:
Since this function is defined in a generic `util` package, and contains _no_ white list pod logic, I think your original name `LoadTokens` (or similar, generic line reading name of your choice) is actually more appropriate